src/main/com/fulcrologic/fulcro/algorithms/denormalize.cljc

```diff
@@ -148,13 +148,13 @@
 (defn db->tree
   "Pull a tree of data from a fulcro normalized database as a tree corresponding to the given query.
 
-  query - EQL
+  query - EQL.
   starting-entity - A map of data or ident at which to start.
   state-map - The overall normalized database from which idents can be resolved.
 
   Returns a tree of data where each resolved data node is also marked with the current
-  *denormalize-time* (dynamically bound outside of this call).  Users of this function that
-  are hydrating the UI should ensure that this time is bound to Fulcro's current internal
+  *denormalize-time* (dynamically bound outside of this call). Users of this function that
+  are hydrating the UI, should ensure that this time is bound to Fulcro's current internal
```

Comment:
I think the comma does not belong here.  If you find it an awkward construction it could be reworded.  How about an active voice "Use `binding` on `*denormalize-time*` to Fulcro's basis time  when using the result in rendering."

Reply:
Yes, I do think that active voice makes more sense for docstrings, kind of sound more contextual.

The comma is kind of a filler for a natural pause in the flow of a sentence, I think. But of course, being from India I'm more exposed towards British sense of flow in the language.

I'll remove the comma, no problem 👍